components/treeview/drag-drop.md

```diff
@@ -28,12 +32,41 @@ To enable the Drag and Drop functionality:
 
 1. Set the `Draggable` parameter of the `<TelerikTreeView>` to `true`
 
-1. Use the `OnDrop` event to handle the drag and drop operations and modify the data source as per your business logic.
+1. Use the Drag events to handle the drag and drop operations and modify the data source as per your business logic.
 
+## OnDragStart Event
+
+The `OnDragStart` event fires when the user starts dragging a node. It provides details for the dragged items and allows you to cancel the event.
```

Comment:
The dragged item can have children, but it's always one, so we may not introduce a confusion here.

```suggestion
The `OnDragStart` event fires when the user starts dragging a node. It provides details for the dragged item and allows you to cancel the event.
```